crates/valence_protocol/src/var_long.rs

```diff
@@ -1,7 +1,11 @@
+#[cfg(target_arch = "x86")]
+use std::arch::x86::*;
+#[cfg(target_arch = "x86_64")]
+use std::arch::x86_64::*;
```

Comment:
You can scope these imports to the body of `encode` so you don't have to use `cfg` here.